librz/core/cmd/cmd_analysis.c

```diff
@@ -1836,23 +1837,27 @@ RZ_IPI RzCmdStatus rz_analysis_syscall_number_handler(RzCore *core, int argc, co
 }
 
 static void syscall_dump(RzSyscallItem *si, bool is_c) {
+	char tmpbuf[32] = { 0 };
+	snprintf(tmpbuf, sizeof(tmpbuf), SYSCALL_NUM_FMT(si->num), si->num);
```

Comment:
use `rz_strf`

Reply:
I tried but it complains then about possible NULL value